src/Button/styles.ts

```diff
@@ -89,6 +96,7 @@ export const getVariantStyles = (variant: VariantType = 'default', theme?: Theme
         backgroundColor: 'btn.danger.disabledBg',
         borderColor: 'btn.danger.disabledBorder',
         '[data-component="ButtonCounter"]': {
+          color: 'btn.danger.disabledText',
```

Comment:
Optional: As this is the same as the disabled button color, we could use `color: inherit` so that matches the parent even when we change the color value for the parent